server/src/test/java/org/elasticsearch/index/mapper/ExternalMapper.java

```diff
@@ -63,6 +63,7 @@
         private BooleanFieldMapper.Builder boolBuilder = new BooleanFieldMapper.Builder(Names.FIELD_BOOL);
         private GeoPointFieldMapper.Builder latLonPointBuilder = new GeoPointFieldMapper.Builder(Names.FIELD_POINT);
         private GeoShapeFieldMapper.Builder shapeBuilder = new GeoShapeFieldMapper.Builder(Names.FIELD_SHAPE);
+        private LegacyGeoShapeFieldMapper.Builder legacyShapeBuilder = new LegacyGeoShapeFieldMapper.Builder(Names.FIELD_SHAPE);
```

Comment:
let's use the non-legacy mapper here too

Reply:
I have this there for backcompat testing. The `build()` method checks index version and uses this for indexes created before 6.6

Reply:
ok thanks for explaining